.github/workflows/docker-build.yml

```diff
@@ -336,8 +345,10 @@ jobs:
           platforms: ${{ inputs.platforms }}
           cache-from: |
             type=local,src=/tmp/outputs/cache/docker
```

Comment:
Does this mean it is using cache from local and  conditionally from github

Reply:
@shambupujar Is that a concern?

Reply:
Resolving as I believe it to be rather help than harmful to have a default cache used always.